packages/venia-concept/src/components/ToastContainer/toast.css

```diff
@@ -0,0 +1,129 @@
+.toast {
+    align-items: start;
+    background-color: white;
+    border-radius: 2px;
+    box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.5);
+    color: rgb(33, 33, 33);
+    display: grid;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 0.8rem;
+    font-weight: lighter;
+    gap: 1rem;
+    grid-template-areas: 'icon message controls';
+    grid-template-columns: min-content 200px auto;
+    grid-template-rows: repeat(auto-fit, 1fr);
+    justify-items: start;
+    line-height: 1.25rem;
+    margin: 0 auto;
+    padding: 1rem;
+    width: 20rem;
+
+    border: 1px solid;
+    border-color: #d1d1d1;
+}
+
+/* TODO: Figure out how to pulse borders for "identical" react trees */
```

Comment:
@jimbo @supernova-at do ya'll have ideas about how to go about this? If a toast currently exists in the DOM and an identical toast is added (same props) Soumya asked that we reset the timeout and "pulse" the border.

This CSS pulses the border when a toast is first added but it does not pulse when I add a second duplicate toast. You can repro by checking the PR out at this state and running the app. You can see in `logo.js` that I add two identical toasts after a timeout, and though the removal timer is reset the pulse does not occur.

Reply:
Handled it by using a random key. 🤷‍♂